rust/src/types.rs

```diff
@@ -2212,13 +2212,9 @@ impl TryFrom<&Var> for &str {
     if var.valueType != SHType_String
       && var.valueType != SHType_Path
       && var.valueType != SHType_ContextVar
-      && var.valueType != SHType_None
     {
-      Err("Expected None, String, Path or ContextVar variable, but casting failed.")
+      Err("Expected String, Path or ContextVar variable, but casting failed.")
     } else {
-      if var.valueType == SHType_None {
-        return Ok("");
-      }
```

Comment:
I understand the reason, but it might break somewhere... Let's keep an eye on it.

Reply:
yea tbh I think it's my fault, some of my old old old code was using this hack....